package/MDAnalysis/analysis/rms.py

```diff
@@ -670,6 +676,9 @@ def _prepare(self):
 
         self._mobile_coordinates64 = self.mobile_atoms.positions.copy().astype(np.float64)
 
+    def _get_aggregator(self):
+        return ResultsGroup(lookup={'rmsd': ResultsGroup.ndarray_sum})
```

Comment:
Unfortunately, this is probably the last thing I'm yet struggling with a bit (surprisingly) -- how to pass exactly the necessary amount of work. I did a few tries but didn't succeed -- mainly due to the fact that `n_frames` and `times` do not lay in `self.results`, but are general attributes of the class, and it's slightly harder to keep track of it.

Since I believe all the rest is fixed now, I'll try to put more effort into it.